manifests/consumer.pp

```diff
@@ -56,7 +56,7 @@
 ) inherits kafka::params {
 
   validate_re($::osfamily, 'RedHat|Debian\b', "${::operatingsystem} not supported")
-  validate_absolute_path($install_dir)
+  #validate_absolute_path($install_dir)
```

Comment:
Please remove this, don't uncomment it. Git keeps the history for us if we need to revert.